src/mon/OSDMonitor.cc

```diff
@@ -7932,13 +7932,12 @@ int OSDMonitor::check_pg_num(int64_t pool, int pg_num, int size, int crush_rule,
       }
       osd_num += osd_ids.size() - out_osd;
     }
-    if (pool >= 0) {   
```

Comment:
Needs `cherry-picked from`. What's weird it lacks `Fixes` too. How is it `main`?